src/views/mixed-content/top-countries-with-more-websites-with-mixed-content/index.js

```diff
@@ -0,0 +1,93 @@
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import React, {useState, useEffect} from 'react';
+import {api} from '../../../client';
+import {ChartContainer} from './elements';
+import {ResponsiveChoropleth} from '@nivo/geo';
+import * as features from './features.json';
+
+
+const TopCountriesWithMoreWebsitesWithMixedContent = () => {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    api
+        .get('/api/mixed-content/top-countries' +
+        '-with-more-government-websites-with-mixed-content')
+        .then((response) => {
+          setData(response.data.result);
+          console.log('fetched');
+          setLoading(false);
+        })
+        .catch((err) => {
+          console.log(err);
```

Comment:
We can handle these errors using a [Snackbar](https://material-ui.com/es/components/snackbars/) to avoid console usage in production

Reply:
Done, I may as well make a custom component, I just wrote the same code in every file.